Agenda/Modules/Summary/SummaryPresenter.swift

```diff
@@ -14,6 +14,9 @@ final class SummaryPresenter {
     private let router: SummaryRouterInput
     private let interactor: SummaryInteractorInput
     
+    /// This array describes what kind of data will be displayed in cells. User selects the data he needs and then we add/remove these `SummaryCell` enum's cases
+    public var cells: [SummaryKind] = [.percentOfSetGoals, .completedGoals, .uncompletedGoals, .allGoals]
+    
```

Comment:
It is interactor's responsibility to prepare the data to view. Replace this code to the interactor